src/liger_kernel/transformers/monkey_patch.py

```diff
@@ -295,18 +322,30 @@ def apply_liger_kernel_to_phi3(
 }
 
 
-def _apply_liger_kernel(model_type: str = "", **kwargs) -> None:
+def _apply_liger_kernel(model: PreTrainedModel = None, model_type: str = "", **kwargs) -> None:
```

Comment:
i would prefer separate this as another api. maybe `apply_liger_kernel_on_instance`. Would love the thoughts from @yundai424 @qingquansong

Reply:
seconded on separating instance-patch to a different API. Otherwise apply_liger_kernel_to_llama(model=<a llama model>) is just a liiitle bit cumbersome because the model itself has told what model family is it 🤔

Reply:
Makes sense, I can make it a separate API